.travis.yml

```diff
@@ -17,18 +17,17 @@ env:
   matrix:
     - PLONE_VERSION=4.3
     - PLONE_VERSION=4.3 DEXTERITY_ONLY=true
-    - PLONE_VERSION=4.3 JQUERY=1.9.1.3
+    - PLONE_VERSION=4.3 VERSIONS=versions:plone.app.jquery=1.9.2
```

Comment:
@hvelarde Really nice, `bin/buildout buildout:eggs+=plone.app.drafts` for example works as well. This is a really nice trick for testing and CI. For example, instead of using `DEXTERITY_ONLY=true` and trying to read an environment variable, you could have `bin/buildout buildout:eggs+=plone.app.contenttypes` in a different job.